bin/test-workflow/start

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eo pipefail
```

Comment:
I think we'll want a README.md file to give a brief overview of what these scripts do, and how to run it (e.g. just run `./start`). This wasn't included in the DoD for the corresponding issue(s), so maybe we can file a follow-up issue for this if that works better.

There's a README.md that we might be able to grab some content:
   https://github.com/cyberark/conjur-oss-helm-chart/blob/master/examples/kubernetes-in-docker/README.md
although this README.md ^^^^ has much more verbage than we need here.

Reply:
Created #314